spec/rails_helper.rb

```diff
@@ -61,5 +61,8 @@
     with.test_framework :rspec
     with.library :rails
   end
+end
 
+RSpec.configure do |config|
+  config.include Rails.application.routes.url_helpers
```

Comment:
Definitely agree that it's a good idea to reference paths in your specs, although I'm not sure you need this config line for that. Do your specs in `user_creation_spec.rb` still work if you remove this line?